system-variables.md

```diff
@@ -1768,6 +1768,14 @@ explain select * from t where age=5;
 3 rows in set (0.00 sec)
 ```
 
+### tidb_opt_skew_distinct_agg <span class="version-mark">New in v6.2.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: No
```

Comment:
Should be YES. The Chinese version needs to be updated too.

Reply:
```suggestion
- Persists to cluster: Yes
```